src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

```diff
@@ -384,6 +384,25 @@ public Constraint Matches<TActual>(Predicate<TActual> predicate)
 
         #endregion
 
+        #region MultipleOf, Even, Odd
+
+        /// <summary>
+        /// Returns a constraint that tests for a number to be a multiple of another.
+        /// </summary>
+        public MultipleOfConstraint MultipleOf(int multiple) => Append(new MultipleOfConstraint(multiple));
+
+        /// <summary>
+        /// Returns a constraint that tests for a number to be even (i.e. a multiple of two)
+        /// </summary>
+        public MultipleOfConstraint Even => MultipleOf(2);
```

Comment:
This would fail for 0, but 0 is considered even.

https://en.wikipedia.org/wiki/Parity_of_zero

Reply:
No it does not. 0 is a multiple of 2. (0 * 2 => 0).
To verify this I added a test to the PR:
```csharp
Assert.That(0, Is.Even);
```

Reply:
Oh I thought I saw that the multiple of constraint had a guard where the input had to be greater than 0. But it was probably looking at the 2, not the 0, my bad.